src/fdrs_gateway_filesystem.h

```diff
@@ -1,34 +1,23 @@
 
-
 #ifdef USE_SD_LOG
 #include <SPI.h>
 #include <SD.h>
 #endif
 #ifdef USE_FS_LOG
 #include <LittleFS.h>
 #endif
-#if defined(USE_SD_LOG) || defined(USE_FS_LOG)
 #include <time.h>
-#endif
-#if defined(USE_SD_LOG) || defined(USE_FS_LOG)
+
+#define SD_MAX_FILESIZE 1024
```

Comment:
SD_MAX_FILESIZE was not previously defined.  Not sure that previously the code would compile without this defined.